mosaic/descriptor.py

```diff
@@ -17,60 +20,119 @@
                        "edgesOnVertex"]
 
 
-class Descriptor:
+def attr_to_bool(attr: str):
+    """Format attribute strings and return a boolean value
     """
```

Comment:
I'd typically format a single-line docstring like this:
```suggestion
    """ Format attribute strings and return a boolean value """
```